src/fsharp/CompileOps.fs

```diff
@@ -2549,6 +2547,56 @@ let GetAutoOpenAttributes ilg ilModule =
 
 let GetInternalsVisibleToAttributes ilg ilModule = 
     ilModule |> GetCustomAttributesOfILModule |> List.choose (TryFindInternalsVisibleToAttr ilg)
+
+let findPrimaryAssembly (data: TcConfigBuilder) =
+    let assemblies =
+        data.referencedDLLs
```

Comment:
We may not have the correct paths at this time because assembly resolution has not happened yet. This means we need to find the primary assembly after assembly resolution.

The problem with that is assembly resolution requires knowledge of a primary assembly... @KevinRansom is that right?